diff --git a/browser/moz.configure b/browser/moz.configure
index 3ea3d88b9360..18ae06c855c4 100644
--- a/browser/moz.configure
+++ b/browser/moz.configure
@@ -11,8 +11,8 @@ imply_option("MOZ_NORMANDY", True)
 imply_option("MOZ_PROFILE_MIGRATOR", True)
 
 
-imply_option("MOZ_APP_VENDOR", "Mozilla")
-imply_option("MOZ_APP_ID", "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}")
+imply_option("MOZ_APP_VENDOR", "Dandelion")
+imply_option("MOZ_APP_ID", "{dd827967-20ec-41ff-8876-0094a32cf3a9}")
 # Include the DevTools client, not just the server (which is the default)
 imply_option("MOZ_DEVTOOLS", "all")
 imply_option("BROWSER_CHROME_URL", "chrome://browser/content/browser.xhtml")
